src/DearImGui.hs

```diff
@@ -818,11 +830,16 @@ setNextWindowSizeConstraints sizeMinRef sizeMaxRef = liftIO do
       with sizeMax $ \sizeMaxPtr -> 
         [C.exp| void { SetNextWindowSizeConstraints(*$(ImVec2 *sizeMinPtr), *$(ImVec2 *sizeMaxPtr)) } |]
 
-
+-- | Set next window collapsed state. call before `begin`
+--
+-- Wraps @ImGui::SetNextWindowCollapsed()@
 setNextWindowCollapsed :: (MonadIO m) => CBool -> ImGuiCond -> m ()
```

Comment:
Use `Bool` and convert manually to a `CBool`.

Reply:
fixed